### ISSUE TITLE
Set node set name for multiple node sets

### ISSUE THREAD

**Issue by SidneyLann** [joshcarty/tfgnn-ogb#8]
hidden = tfgnn.keras.layers.ReadoutFirstNode(node_set_name="paper")(hidden)
    output = tf.keras.layers.Dense(num_classes, activation="softmax")(hidden)
    return model.NodeClassificationModel(
        input, output, target_node="paper", label_name="label"
    )

In this example, there is only one node set paper, how to set the node_set_name and target_node for above code in multiple node sets app?
